### PR TITLE
allow using alias in path

### PR DESCRIPTION
**Pull request by ruifortes**
loader context doesn't have access to global config now so I used the options object.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,12 +31,23 @@ module.exports = function(source) {
...
+  var alias = loaderUtils.getOptions(this).alias
+  
   function replacer(match, fromStatement, obj, quote, filename) {
     var modules = [];
     var withModules = false;
 
     if (!filename.match(/\*/)) return match;
+    
+    if(alias) {	
+			Object.entries(alias).some(([alias, repl]) => {
```
Comment (fred104) [inline index.js:44]: We prefer two spaces here.

File: index.js [hunk 2]
```diff
@@ -30,12 +31,23 @@ module.exports = function(source) {
   var resourceDir = path.dirname(this.resourcePath);
 
   var nodeModulesPath = walkUpToFindNodeModulesPath(resourceDir);
-
+  
+  var alias = loaderUtils.getOptions(this).alias
```
Comment (fred104) [inline index.js:35]: Missing semicolon here.

**Review by fred104 — CHANGES_REQUESTED** [review]
Hi Rui, thanks for your kind PR. Can you help add some tests and modify the README.md?

**Comment by soluml** [comment]
@ruifortes, @fred104 - Ran into this issue today, would love to be able to use Alias's with this loader.  I see this is a year old issue, any chance of it being resolved? Can I help? Seems like the PR just needs some linting changes and and updated package-lock.

**Comment by fred104** [comment]
@soluml If you can help, that would be great. 👍

**Comment by soluml** [comment]
@fred104, @ruifortes - Done in PR #4

**Pull request merged**